src/main/java/seedu/address/logic/parser/DeleteTagCommandParser.java

```diff
@@ -0,0 +1,33 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.logic.commands.DeleteTagCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Parses input arguments and creates a deleteTagCommand object.
+ */
+public class DeleteTagCommandParser implements Parser<DeleteTagCommand> {
+    /**
+     * Parses the given {@code String} of arguments in the context of the DeleteTagCommand
+     * and returns an DeleteTagCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public DeleteTagCommand parse(String args) throws ParseException {
+        try {
+            int lengthOfUserInput = args.split(" ").length;
+            if (lengthOfUserInput > 2 || lengthOfUserInput <= 1) {
```

Comment:
is it the same as "if (lengthOfUserInput != 2)" ?
did you consider trimming the args first?